lib/modules/platform/gerrit/index.spec.ts

```diff
@@ -252,36 +274,50 @@ describe('modules/platform/gerrit/index', () => {
         `the change should be created automatically from previous push to refs/for/source`,
       );
     });
+
+    it('createPr() - found existing but not created in the last 5 minutes => rejects', async () => {
+      const change = partial<GerritChange>({
+        _number: 123456,
+        // 6 minutes ago
+        created: new Date(Date.now() - 6 * 60 * 1000).toISOString(),
```

Comment:
use luxon, it's much easier to read 

you should use static dates and use vitest fake timers